test/php/library/Icinga/Web/FormTest.php

```diff
@@ -120,8 +120,6 @@ public function testWhetherAnExplicitlySetRedirectUrlIsUsedForRedirection()
      */
     public function testWhetherACsrfCounterMeasureIsBeingAdded()
     {
-        Mockery::mock('alias:Icinga\Web\Session')->shouldReceive('getSession->getId')->andReturn('1234567890');
```

Comment:
Why is this removed? Because the chart tests initialize the session? Tests shouldn't rely on each other. This **must** be solved differently.

Reply:
The session is now initialized in `BaseTestCase`. Hence this might not be required anymore.